### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Access Token_Screen Reader: NVDA is reading whole sentence as two parts after access token.

### ISSUE THREAD

**Issue by jhkimnew** [Microsoft/IIS.Administration#273]
Copied from https://github.com/microsoft/IIS.WebManager/issues/155

Environment:
**URL:**https://localhost:55539/security/tokens#
Browser Details: Chrome(68.0.3440.106)
Screen Reader: NVDA
MAS ID Violated: 1.3.2

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till "Skip This" button and Press Enter.
4.Tab till "Get Access Token" link and Press Enter
5.Tab till "Create Access Key" button and Press Enter.
6.Tab till "Create" button and press enter.
7.Navigate to non interactive elements using down arrow.
8.Verify how NVDA is reading message provided below access token.

Actual Result:
NVDA is reading whole sentence as two parts after access token.

Expected Result:
NVDA should read whole sentence at a time instead of reading by parts.

User Impact:
Screen reader users may think the whole sentence as two different sentences which is not correct. It causes inconvenience to the user.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

**Comment by yaqiyang** [comment]
Closed. New features are not going to be implemented in the near future
